src/app/cli/CommandLineInterface.java

```diff
@@ -63,15 +92,280 @@ public void launch(){
+                        System.out.println("You do not have a collection with that name.");
+                        continue;
+                    }
+                    Collection selected_coll = collections_list.get(0);
+
+                    if(collections_list.size() > 1) {
+                        System.out.println("Which collection would you like to add the game to (enter the number)?");
+                        for(int i = 0; i < collections_list.size(); ++i) {
+                            Collection curr_coll = collections_list.get(i);
+                            System.out.println(i+1 + ".\tName: " + curr_coll.coll_name() + "\n\t# of games: " +
+                                    curr_coll.games().length + "\n\tPlay time: " +
+                                    app.total_playtime_collection(curr_coll) + "\n");
+                        }
+                        input = in.nextLine();
+                        int input_to_int = Integer.parseInt(input);
+                        selected_coll = collections_list.get(input_to_int-1);
+                    }
+                    Game[] game_list = app.search_game_name(tokens[2]);
+
+                    if(game_list.length == 0) {
+                        System.out.println("There is not a game with that name.");
+                        continue;
+                    }
+
+                    Game selected_game = game_list[0];
+                    if(game_list.length > 1) {
+                        System.out.println("Which game would you like to add (enter the number)?");
+                        for(int i = 0; i < game_list.length; ++i) {
+                            Game curr_game = game_list[i];
+                            System.out.println(i+1 + ".\tName: " + curr_game.title() + "\n\tPlay time: " +
+                                    curr_game.playtime());
+                        }
+                        input = in.nextLine();
+                        int input_to_int = Integer.parseInt(input);
+                        selected_game = game_list[input_to_int-1];
+                    }
+
+                    Platform[] user_platforms = app.get_platforms();
+
+                    Platform[] game_platforms = app.get_game_platforms(selected_game);
+
+                    boolean platform_match = false;
+
+                    for (Platform user_platform : user_platforms) {
+                        for (Platform game_platform : game_platforms) {
+                            if (user_platform.name().equals(game_platform.name())) {
+                                platform_match = true;
+                                break;
+                            }
+                        }
+                    }
+
+                    if(!platform_match) {
+                        System.out.println("*WARNING*\nYou do not own the platform this game is on. Proceed (y/n)?");
+                        input = in.nextLine();
+
+                        if(input.equals("n")) {
```

Comment:
doesn't really really matter but getting first character of input would be more robust (or even a y/n prompt method)
`if (input.toLowerCase().charAt(0) == 'n') {`